c.html.markdown

```diff
@@ -162,6 +162,8 @@ int main (int argc, char** argv)
 
   // Strings are just arrays of chars terminated by a NULL (0x00) byte,
   // represented in strings as the special character '\0'.
+  // NULL characters are vital to functions that operate on strings,
```

Comment:
Reads a bit odd when together with the next sentence. Maybe combine them like "We don't have to ..., but if you're constructing one in code, be sure to include them"? Also, while we're on the subject, definitely mention that you should have an extra byte when allocating space for strings. That seems to trip people up as well.